python/sglang/global_config.py

```diff
@@ -27,7 +27,7 @@ def __init__(self):
         # Runtime constants: others
         self.num_continue_decode_steps = 10
         self.retract_decode_steps = 20
-        self.flashinfer_workspace_size = 192 * 1024 * 1024
+        self.flashinfer_workspace_size = 384 * 1024 * 1024
```

Comment:
Yeah, we truly need this large work space... otherwise the chunked prefill would probably crash.